python/ray/air/_internal/device_manager/utils.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from typing import Optional
+
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
+from ray.air._internal.device_manager.hpu import HPUTorchDeviceManager
+from ray.air._internal.device_manager.npu import (
+    NPU_TORCH_PACKAGE_AVAILABLE,
+    NPUTorchDeviceManager,
+)
+from ray.air._internal.device_manager.nvidia_gpu import CUDATorchDeviceManager
+from ray.air._internal.device_manager.torch_device_manager import TorchDeviceManager
+
+logger = logging.getLogger(__name__)
+
+
+SUPPORTED_ACCELERATOR_TORCH_DEVICE_MANAGER = {
+    "GPU": CUDATorchDeviceManager,
+    "HPU": HPUTorchDeviceManager,
+    "NPU": NPUTorchDeviceManager,
+}
+
+
+def try_register_torch_accelerator_module() -> None:
```

Comment:
what's the purpose of this? I do not want to raise if someone doesn't `torch_npu` installed.

Reply:
It is used to import/register relevant PyTorch extensions required by accelerators. Currently, for users who do not use NPU or HPU, no errors will occur. The error message has been optimized to avoid misunderstandings.

Reply:
can we do this on demand rather than at the file level in `torch/config.py`? The auto global import seems a bit weird to me.